src/status_im/native_module/core.cljs

```diff
@@ -380,3 +380,7 @@
 (defn deactivate-keep-awake []
   (log/debug "[native-module] deactivateKeepAwake")
   (.deactivateKeepAwake ^js (status)))
+
+(defn reset-keyboard-input [input selection]
+  (log/debug "[native-module] resetKeyboardInput")
+  (.resetKeyboardInputCursor ^js (status) input selection))
```

Comment:
we don't have the implementation in objc. ? how will it wiork on ios ?

Reply:
It won't be called in objc as it is not needed there. It is not needed even on all Androids. I will add some comments now

Reply:
added comment